src/pages/Clusters.js

```diff
@@ -52,88 +129,171 @@ const Clusters = () => {
+      disks && h(SimpleFlexTable, {
+        rowCount: filteredDisks.length,
+        columns: [
+          {
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
+            cellRenderer: ({ rowIndex }) => {
+              const disk = filteredDisks[rowIndex]
+              const multiple = _.remove({ status: 'Deleting' }, disksByProject[disk.googleProject]).length > 1
+              return h(Fragment, [
+                disk.googleProject,
+                disk.status !== 'Deleting' && multiple && h(TooltipTrigger, {
+                  content: 'This billing project has multiple active persistent disks. Only one will be used.'
+                }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.warning() } })])
+              ])
+            }
+          },
+          {
+            size: { basis: 120, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'size', onSort: setDiskSort }, ['Size (GB)']),
+            cellRenderer: ({ rowIndex }) => {
+              const disk = filteredDisks[rowIndex]
+              return disk.size
+            }
+          },
+          {
+            size: { basis: 150, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'status', onSort: setDiskSort }, ['Status']),
+            cellRenderer: ({ rowIndex }) => {
+              const disk = filteredDisks[rowIndex]
+              return disk.status
+            }
+          },
+          {
+            size: { basis: 250, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'created', onSort: setDiskSort }, ['Created']),
+            cellRenderer: ({ rowIndex }) => {
+              return makeCompleteDate(filteredDisks[rowIndex].auditInfo.createdDate)
+            }
+          },
+          {
+            size: { basis: 250, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'accessed', onSort: setDiskSort }, ['Last accessed']),
+            cellRenderer: ({ rowIndex }) => {
+              return makeCompleteDate(filteredDisks[rowIndex].auditInfo.dateAccessed)
+            }
+          },
+          {
+            size: { basis: 250, grow: 0 },
+            headerRenderer: () => {
+              return h(Sortable, { sort: diskSort, field: 'cost', onSort: setDiskSort }, [`Cost / month (${formatUSD(totalDiskCost)} total)`])
+            },
+            cellRenderer: ({ rowIndex }) => {
+              return formatUSD(persistentDiskCostMonthly(filteredDisks[rowIndex]))
+            }
+          },
+          {
+            size: { basis: 50, grow: 0 },
+            headerRenderer: () => null,
+            cellRenderer: ({ rowIndex }) => {
+              const { id, status } = filteredDisks[rowIndex]
+              // TODO PD: there should be some way of identifying which disk is connected to which runtime
```

Comment:
Ah, interesting. One quick thought that doesn't quite address this is to add some icons to at least indicate whether or not a runtime has a disk attached and whether or not a disk is attached to a runtime. Currently the only way to see that it either in the runtime delete modal or whether or not the disk delete button is disabled.

Reply:
Yeah, one potential idea is to add icons, and let people click on them to highlight the linkages between rows. Not sure if that's a bit too weird though.